crates/triangulation/src/point.rs

```diff
@@ -305,6 +306,14 @@ impl Triangle {
     pub fn new(x: Index, y: Index, z: Index) -> Self {
         Triangle { x, y, z }
     }
+
+    pub fn shifted_by(&self, shift: Index) -> Self {
```

Comment:
Instead of shift_by perhaps simply `shift`

Reply:
For me, shift suggests doing it in place.

Reply:
Since most folks will be coming from a Python background where a Builder pattern is less explicit than Rust, I think the docstring will reduce the assumption that one is shifting in place.

Reply:
added docstring